tests/helpers.py

```diff
@@ -252,3 +252,16 @@ def fetch_csv(type: ReportType, output_dir, report_id):
         r = session.get(f"https://pol.is/api/v3/reportExport/{report_id}/{type.value}.csv")
         f.write(r.text)
     return f
+
+def convert_ids_to_strings(data_list: list) -> list:
+    """Convert integer statement and participant IDs to string IDs"""
+    for item in data_list:
+        if "participant_id" in item:
+            item["participant_id"] = str(item["participant_id"])
+        if "statement_id" in item:
+            item["statement_id"] = str(item["statement_id"])
+    return data_list
+
+def convert_participant_ids_to_strings(participant_ids: list[int]) -> list[str]:
+    """ Convert a list of participant IDs (int) to a list of string IDs"""
+    return [str(pid) for pid in participant_ids]
```

Comment:
this should convert the numeric pid to one that _can't_ be converted back. Maybe like this:

```suggestion
    return ["p{}".format(str(pid)) for pid in participant_ids]
```